### PR TITLE
security/tailscale dashboard widget, ipv6 advertised routes

### LINKED ISSUE #4412: security/tailscale: support ipv6 advertised subnets
**Important notices**
Before you add a new report, we ask you kindly to acknowledge the following:

- [x ] I have read the contributing guide lines at https://github.com/opnsense/plugins/blob/master/CONTRIBUTING.md
- [ x] I have searched the existing issues, open and closed, and I'm convinced that mine is new.
- [ ] When the request is meant for an existing plugin, I've added its name to the title.

**Is your feature request related to a problem? Please describe.**
N/A

**Describe the solution you'd like**
Support ipv6 advertised subnets for tailscale

### PR DESCRIPTION
**Pull request by sheridans**
Adds Tailscale widget to dashboard.
Allow IPv6 advertised subnets, closes #4412
Show online status for peers in status page

### CODE REVIEW

File: security/tailscale/pkg-descr [hunk 1]
```diff
@@ -6,6 +6,13 @@ https://tailscale.com/
 Plugin Changelog
 ================
 
+1.0_1
```
Comment (fichtner) [inline security/tailscale/pkg-descr:9]: while I don't mind the _1 choice we don't do hotfix changelogs here. Either move to 1.0 or 1.1 for clarity

"initial release" is hardly ever expandable, but also difficult to explain in full

File: security/tailscale/src/opnsense/mvc/app/views/OPNsense/Tailscale/status.volt [hunk 2]
```diff
@@ -8,7 +8,7 @@
             return (str + '').replace(/([^>\r\n]?)(\r\n|\n\r|\r|\n)/g, '$1' + breakTag + '$2');
         }
 
-	function updateNetInfo() {
+	    function updateNetInfo() {
```
Comment (fichtner) [inline security/tailscale/src/opnsense/mvc/app/views/OPNsense/Tailscale/status.volt:11]:
```suggestion
	function updateNetInfo() {
```

File: security/tailscale/src/opnsense/www/js/widgets/Metadata/Tailscale.xml [hunk 3]
```diff
@@ -0,0 +1,25 @@
+<metadata>
+    <Tailscale>
+        <filename>Tailscale.js</filename>
+        <endpoints>
+            <endpoint>/api/tailscale/service/status</endpoint>
+            <endpoint>/api/tailscale/status/status</endpoint>
```
Comment (fichtner) [inline security/tailscale/src/opnsense/www/js/widgets/Metadata/Tailscale.xml:6]: can you check with "make lint-acl".. relatively sure it will complain

Comment (sheridans) [reply security/tailscale/src/opnsense/www/js/widgets/Metadata/Tailscale.xml:6]: root@OPNsense:/usr/plugins/security/tailscale # make lint-acl
Did not find ACLBIN, please provide a core repository
*** Error code 1

Comment (sheridans) [reply security/tailscale/src/opnsense/www/js/widgets/Metadata/Tailscale.xml:6]: it did, fixed.

**Comment by fichtner** [comment]
ok, ready? :)

**Comment by sheridans** [comment]
Make it so :)

**Comment by fichtner** [comment]
Merged, thanks!

**Pull request merged**